packages/processors/test/allo/handlers/poolFunded.handler.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+
+import type { ChainId, ILogger, ProcessorEvent, Token } from "@grants-stack-indexer/shared";
+import { IPricingProvider } from "@grants-stack-indexer/pricing";
+import { IRoundReadRepository, Round } from "@grants-stack-indexer/repository";
+import { TOKENS, UnknownToken } from "@grants-stack-indexer/shared";
+
+import { calculateAmountInUsd } from "../../../src/helpers/index.js";
+import { PoolFundedHandler } from "../../../src/processors/allo/handlers/index.js";
+
+function createMockEvent(
+    overrides: Partial<ProcessorEvent<"Allo", "PoolFunded">> = {},
+): ProcessorEvent<"Allo", "PoolFunded"> {
+    return {
+        blockNumber: 116385567,
+        blockTimestamp: 1708369911,
+        chainId: 10 as ChainId,
+        contractName: "Allo",
+        eventName: "PoolFunded",
+        logIndex: 123,
+        srcAddress: "0x1133eA7Af70876e64665ecD07C0A0476d09465a1",
+        params: {
+            poolId: "1",
+            amount: "100",
+            fee: "10",
+        },
+        transactionFields: {
+            hash: "0xtransactionhash",
+            transactionIndex: 5,
+            from: "0xsenderaddress",
+        },
+        ...overrides,
+    };
+}
```

Comment:
there's a shared `createMock` function in `event.mock.ts` file

Reply:
I will skip this, since is not that straight forward to implement it for Allo events, will add it as a low priority tech debt